src/components/menu/menu.spec.ts

```diff
@@ -320,6 +353,138 @@ describe('MenuComponent', () => {
         expect(component.trigger.menu._isFocused$.value).toBeFalsy();
         expect(component.subMenuTrigger.menu._isHovering$.value).toBeFalsy();
         expect(component.subMenuTrigger.menu._isFocused$.value).toBeFalsy();
+    });
+});
 
+@Component({
+    selector: 'app-menu-test',
+    template: `
+        <div class="btn-group" *ngIf="!destroyed">
+            <button
+                type="button"
+                class="btn button-secondary dropdown-toggle"
+                [uxMenuTriggerFor]="menu"
+            >
+                Actions
+                <ux-icon name="down" class="dropdown-icon-inline"></ux-icon>
+            </button>
+        </div>
+
+        <ux-menu #menu>
+            <button uxMenuItem>Item One</button>
+            <ux-menu-divider></ux-menu-divider>
+            <button
+                uxMenuItem
+                #subMenuTrigger="ux-menu-trigger"
+                [uxMenuTriggerFor]="subMenu"
+            >
+                Item Two
+            </button>
+            <button uxMenuItem>Item Three</button>
```

Comment:
All we need is a menu with ngFor, so get rid of all this stuff and just add one collection of buttons.